content/en/docs/setup/keadm.md

```diff
@@ -15,83 +15,135 @@ Please refer [kubernetes-compatibility](https://github.com/kubeedge/kubeedge#kub
 
 ## Limitation
 
-- Currently support of `keadm` is available for Ubuntu and CentOS OS. RaspberryPi supports is in-progress.
 - Need super user rights (or root rights) to run.
-- The subcommand `keadm beta` is launched since v1.10.0. If you would like to use it, please use v1.10.0 or above of keadm.
+
 
 ## Install keadm
 
-Run the command below to one-click install `keadm`.
+There're three ways to download a `keadm` binary
+
+- Download from github release.
+  
+    Now KubeEdge github officially holds three arch releases: amd64, arm, arm64. Please download the right arch package according to your platform.
+    ```shell
+    wget https://github.com/kubeedge/kubeedge/releases/download/v1.12.0/keadm-v1.12.0-linux-amd64.tar.gz
+    tar -zxvf keadm-v1.12.0-linux-amd64.tar.gz
+    cp keadm-v1.12.0-linux-amd64/keadm/keadm /usr/local/bin/keadm
+    ```
```

Comment:
I think we can delete the cmd above? Let users to download self.

Reply:
i think this can be kept, users can download it using the example cmd easily
other comments all addressed